src/vs/workbench/electron-browser/workbench.ts

```diff
@@ -343,7 +351,14 @@ export class Workbench extends Disposable implements IPartService {
 		serviceCollection.set(IListService, this.instantiationService.createInstance(ListService));
 
 		// Context Menu
-		serviceCollection.set(IContextMenuService, new SyncDescriptor(ContextMenuService));
+		const hasCustomTitle = this.getCustomTitleBarStyle() === 'custom';
+
+		// Use themable context menus when custom titlebar is enabled to match custom menubar
+		if (isWindows && hasCustomTitle) {
+			serviceCollection.set(IContextMenuService, new SyncDescriptor(HTMLContextMenuService, undefined, undefined, this.notificationService, this.contextViewService));
```

Comment:
@sbatten the 2x `undefined` seems wrong

Reply:
I added the telemetryservice, not sure if im supposed to pass something for the html container here. its supposed to be the context menu container but we don't have that yet, undefined/null seems to create it on the fly